assets/stylesheets/_components.scss

```diff
@@ -370,3 +370,12 @@
 @import 'signup/validation-fieldset/style';
 @import 'support/support-user/style';
 @import 'vip/vip-logs/style';
+
+// ==========================================================================
```

Comment:
Thoughts on declaring external dependency imports here? Considered creating a separate `_vendor.scss` file instead.

Also a bit unsure about reaching into `node_modules` as I've done here, but it seems to work fine.